components/admin_console/team_channel_settings/channel/details/channel_moderation.tsx

```diff
@@ -340,7 +369,7 @@ export default class ChannelModeration extends React.Component<Props> {
                                 </tr>
                             </thead>
                             <tbody>
-                                {this.props.channelPermissions && this.props.channelPermissions.map((entry) => {
+                                {channelPermissions && channelPermissions.map((entry) => {
```

Comment:
Out of curiousity, what about trying ...

```suggestion
                                {channelPermissions?.map((entry) => {
```

... with the optional chaining instead?

Reply:
Thanks for that catch. I have changed it so it uses optional chaining.

Reply:
😢 Just when I thought all was going well, ESLint started complaining that 
```
  372:54  error  'map' is not defined  no-undef
```

Reply:
I ended up not making this change since I was not able to figure out how to make it pass the ESlint. Maybe our ESLint is out of date or something since these two are the equivalent? I'm not 100% sure.

Reply:
@devinbinnie, @hahmadia and I were experimenting here. Any idea if this is supposed to be possible (seems to work when running locally) and if so, how we get eslint to behave?

Reply:
@hahmadia, no need to hold up merging for this, we can always revisit later. :)